src/main/resources/org/jvnet/hudson/plugins/collapsingconsolesections/CollapsingSectionNote/script.js

```diff
@@ -152,6 +85,16 @@ document.addEventListener("DOMContentLoaded", function () {
         }
     }
 
+    function handleSidePanelChanges() {
+        const sidePanel = document.getElementById("side-panel");
+        const collapsePanel = sidePanel.children.namedItem("console-section-container");
```

Comment:
When the job has no section, there is no `div#console-section-container` and `namedItem()` returns `null`. That still trigger the observer and the code enters an infinite loop cause the container can never be found.

Was reported at https://phabricator.wikimedia.org/T378864 after I have deployed it.